h2olog

```diff
@@ -719,9 +758,7 @@ def load_common_fields(hsh, line):
     fully_qualified_type = line.type.replace("__", ":", 1)
     hsh["type"] = fully_qualified_type
 
-    master_conn_id = line.master_conn_id
-    if master_conn_id != 0:
-        hsh["master_conn_id"] = master_conn_id
+    hsh["master_conn_id"] = line.master_conn_id
```

Comment:
This might look nicer if you remove the empty line above (L760), and pack the lines together.

Reply:
Sure! bbc3cc817cb98c5eaed720bfeced14af235fdec6